src/standalone/Vegetation/autotrophic_respiration.jl

```diff
@@ -0,0 +1,119 @@
+export AutotrophicRespirationParameters, AutotrophicRespirationModel
+
+abstract type AbstractAutotrophicRespirationModel{FT} <: AbstractCanopyComponent{FT} end
+
+"""
+    AutotrophicRespirationParameters{FT<:AbstractFloat}
+
+The required parameters for the autrophic respiration model.
+$(DocStringExtensions.FIELDS)
+"""
+struct AutotrophicRespirationParameters{FT <: AbstractFloat}
+    "Mean leaf nitrogen concentration (kg N (kg C)-1)"
+    nm::FT
+    "Carbon content of roots (?)" # not sure about units
+    Rc::FT
+    "Carbon content of stems (?)"
+    Sc::FT
+    "Specific leaf density (kg C m-2 [leaf])"
+    σl::FT
```

Comment:
σl = 0.05 kg C m-2 (Osborne et al., 2015)

Osborne, T., Gornall, J., Hooker, J., Williams, K., Wiltshire, A., Betts, R., & Wheeler, T. (2015). JULES-crop: a parametrisation of crops in the Joint UK Land Environment Simulator. Geosci. Model Dev, 8, 1139–1155. https://doi.org/10.5194/gmd-8-1139-2015